tests/pytests/functional/states/test_docker_network.py

```diff
@@ -220,10 +220,15 @@ def test_present_with_containers(network, docker, docker_network, container):
 
 
 @pytest.mark.parametrize("reconnect", [True, False])
-def test_present_with_reconnect(network, docker, docker_network, container, reconnect):
+def test_present_with_reconnect(
+    network, docker, docker_network, container, reconnect, grains
+):
     """
     Test reconnecting with containers not passed to state
     """
+    if grains["os_family"] == "Suse":
+        pytest.skip("This test is failing for SUSE family")
```

Comment:
Should we skip this for eternity? Why can't we fix the cause of the test failure?

Reply:
I'm not really sure why exactly this is failing but the error I get is:

```
Failed to create temp network for comparison: Error 500: operation not supported
```

Looking at `docker` service logs in DEBUG:

```
level=debug msg="FIXME: Got an API for which error does not match any expected type!!!" error="operation not supported" error_type=syscall.Errno module=api
level=error msg="Handler for POST /v1.44/networks/create returned error: operation not supported"
level=debug msg="FIXME: Got an API for which error does not match any expected type!!!" error="operation not supported" error_type=syscall.Errno module=api
```

I got the same using SLE15SP4 version of "docker": `24.0.7-ce` and also with latest in Factory: `25.0.3-ce` (but always testing on top of SLE15SP5). Not sure if this is related to the OS, to our docker packages, or really upstream issue.